lib/peerconnection.js

```diff
@@ -49,15 +67,15 @@ function RTCPeerConnection() {
   };
 
   pc.onsignalingstatechange = function onsignalingstatechange() {
-    self.dispatchEvent({ type: 'signalingstatechange' });
+    self.dispatchEvent({ type: 'signalingstatechange', srcElement: self.getSrcElement() });
```

Comment:
Hmm, why wouldn't we just pass `this` as `srcElement`?

Reply:
I have made the changement